test-infra/prow/config.yaml

```diff
@@ -143,6 +143,15 @@ periodics:
           projects/rhel-cloud/global/images/family/rhel-7,\
           projects/freebsd-org-cloud-dev/global/images/family/freebsd-11-2,\
           projects/freebsd-org-cloud-dev/global/images/family/freebsd-12-0-snap"
+      - "-var:alias_ips=\
```

Comment:
I'm not a fan of having to specify a list of alias ips as long as the list of images we test, it seems very cumbersome and something we are very likely to mess up and forget when we add new images. Is the reason for these addresses simply so we can run these tests at the same time? If that's the case I'd say it is probably better to add a new type of lock to the test runner. A duplicate test lock or something, I think a new field something like this for these tests:

"CustomProjectLock": "alias-ip"

Setting this would lock the project with this tag so only one of this type of test will run in the project at a time. If this would work for this set of tests and not require this list of ips here I'll work on adding the additional locking functionality

Reply:
Well, I don't think that would solve the problem actually: Some IP still needs to be given (even if it's just 1). There are always a lot of IP space on the subnetwork so I didn't think it was a concern.

Additionally, I added a check to guarantee that the `len(images) == len(alias_ips)` ( https://github.com/GoogleCloudPlatform/compute-image-tools/pull/455/files#diff-8c48497c3da51f59e852bb3a7a93cea5R8 , ok, it's not really a check but you will see no test at all if it's inconsistent. That's something...).

Another idea is to change the `images` list to have the mapping of the alias_ip explicitly per line (then I'd rewrite the splitting, but at least it'd be 1 list, which is much harder to make mistakes).

Or do you rather prefer having that lock? I'll have to investigate that and I'd ask you to supply the alias-ip that can be used.

Reply:
I still expect us to need to specify at least 1 ip to use, its just the list of IPs that must match the length of the list of images I don't like. Especially if we are running 2 or more test suites with overlapping IP ranges. 

I'm fine with giving a list to use, we could do something fancy and have the lock use the IP as the key so that we could give multiple test suites the same list of IPs (could be more or less than the number of images) and they could all safely run in the same project

Reply:
ok, so there will be a list of IPs that could be used and a lock mechanism to restraint its availability. Understood. I'll take a close look on implementing that

Reply:
We can add a rand function to the template that takes a list and returns a random entry

Reply:
I can add the lock if you want, you can just modify the PR to assume that CustomProjectLock exists.

Something like:
{{$alias_ip := rand $alias_ips -}}
...
...
"test-network [{{$image}}]": {
  "Path": "./network/network.wf.json",
  "Vars": {
     "source_image": "{{$image}}",
    "alias_ip": "{{$alias_ip}}"
  },
  "CustomProjectLock": "alias-ip-{{$alias_ip}}"
},

Reply:
alright. Will do then